charts/cloudzero-agent/values.yaml

```diff
@@ -47,10 +47,18 @@ prometheusConfig:
     # -- Any items added to this list will be added to the Prometheus scrape configuration.
     additionalScrapeJobs: []
 
-kube-state-metrics:
-  enabled: false
+kubeStateMetrics:
```

Comment:
does this work if `alias` is not set in the `dependencies` section? I would have thought that would need to be set. but maybe I'm just not up to date